blueprint/blueprint-core/src/main/java/org/apache/aries/blueprint/container/BlueprintDomainCombiner.java

```diff
@@ -38,7 +38,7 @@ public static AccessControlContext createAccessControlContext(BundleContext bund
 
     public ProtectionDomain[] combine(ProtectionDomain[] arg0,
                                       ProtectionDomain[] arg1) {
-        return new ProtectionDomain[] { new BlueprintProtectionDomain(bundleContext) };
```

Comment:
❓ Any reason we can't _add_ the current domains (`arg0`) to the `BlueprintProtectionDomain()` instead of just returning one or the other? Would sounds a little less drastic to me.

Reply:
BlueprintProtectionDomain has no codeBase. To a security manager, that looks like core Java code and isn't checked.

Reply:
would a better fix for the future be to make the BlueprintProtectionDomain call the other ProtectedDomain constructor that does result in a ProtectionDomain that consults the policy file?

Reply:
Assuming we need to continue using this for an extended period of time, that might be a more elegant solution.